pom.xml

```diff
@@ -126,6 +126,12 @@
                 <scope>test</scope>
             </dependency>
             <dependency>
+                <groupId>org.glassfish.hk2</groupId>
+                <artifactId>hk2-utils</artifactId>
+                <version>2.4.0-b32</version>
+                <scope>test</scope>
+            </dependency>
```

Comment:
Ok, I think we can push this up into elide-integration-test, and remove it from here. Just, tried it and I can make this commit.